modules/user_accounts/test/TestPlan.md

```diff
@@ -59,11 +59,18 @@ When creating or editing a user: (subtest: `edit_user`)
     changes. 
 25. When editing an existing user: Clicking on the 'User Account' breadcrumb takes you to the User Account page
     without saving any changes to the user profile.
+    
+### Testing permissions:
+
+26. r
+1. r
+1. r
+1. r
```

Comment:
@ridz1208 The Testing permission points look weird.

Reply:
ha

Reply:
ha